packages/dex/src/resp_delivery/adapter.rs

```diff
@@ -0,0 +1,20 @@
+use sdk::cosmwasm_std::{Binary, Deps, Env};
+
+use crate::{response::Result, Handler};
+
+pub trait DeliveryAdapter<H, R>
```

Comment:
What's the idea behind the trait is it's not exposed and only implemented once?

Reply:
The aim is to abstract the delivery of response be it via `on_response` or `on_open_ica`. Tha latter follows in the next commits